simulator/README.md

```diff
@@ -31,12 +31,16 @@ From LabVIEW project:
 
 Go to the `simulator/` directory, find `ComponentSimulator.vi`, press right click on it and select **Run**.
 
-Check:
+Please check:
 
 - cmdServerPort = 8888
 - telServerPort = 8889
 - serverTimeout (ms) = 100
 - serverBytesToRead = 32768
+- maxSizeQueueTel = 1000
+- EnqueueOptions = lossy enqueue
+- timeoutQueueTel = 50
```

Comment:
Maybe `timeoutQueueTel (ms) = 50` if I am right for the unit. Thanks!

Reply:
Done. Thanks.